troubleshooting.md

```diff
@@ -55,41 +35,54 @@ the Power of Puppet quest: **admin/puppetlabs**)
 
 ### I can't find the VM password
 
-The password to log in to the VM is generated randomly and will be displayed on
+The Learning VM's password is generated randomly and will be displayed on
 the splash page displayed on the terminal of your virtualization software when
```

Comment:
"will be displayed on the splash page displayed on the" -> Suggest replacing first "displayed" with "shown" or equivalent to avoid repeating "displayed" closely in the same sentence.